pandas/io/formats/printing.py

```diff
@@ -382,7 +382,11 @@ def best_len(values: List[str]) -> int:
         summary = f"[{first}, {last}]{close}"
     else:
 
-        if n > max_seq_items:
+        if max_seq_items == 1:
```

Comment:
do we have a case of max_seq_items==0 ? (I think this is a valid value), < 0 is restricted i am pretty sure?

Reply:
Yes we do. Max_seq_items ==0 implies unlimited if I'm not mistaken